codex-rs/core/tests/live_agent.rs

```diff
@@ -1,3 +1,5 @@
+#![expect(clippy::expect_used)]
```

Comment:
In this file and `live_cli.rs`, I believe `#![allow(clippy::unwrap_used)]` is down below, so should it move to the top in both?